raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -206,6 +245,27 @@ private static Integer parseVoterId(String idString) {
         }
     }
 
+    private static Set<String> parseProcessRoles(List<String> processRoles, Map<Integer, AddressSpec> voterConnections, int nodeId) {
+        Set<String> distinctRoles = new HashSet<>();
+        for (String role : processRoles) {
+            switch (role) {
+                case "broker":
+                    distinctRoles.add("BrokerRole");
```

Comment:
Why don't we use the `ProcessRole` as before?

Reply:
Fixed. See my previous comment.